h2o-py/h2o/model/model_base.py

```diff
@@ -454,6 +454,44 @@ def feature_interaction(self, max_interaction_depth=100, max_tree_depth=100, max
             
             return json['feature_interaction']
         print("No calculation available for this model")
+        
+        
+    def h(self, frame, variables):
+        """
+        Calculates Friedman and Popescu's H statistics, in order to look for interactions among variables in h2o gradient-boosting models
+        
+        NaN is returned if a computation is spoiled by weak main effects and rounding errors.
```

Comment:
This looks like a copy paste from sklearn documentation.

Reply:
yes, probably I could try to reformulate